### ISSUE TITLE
input "QuickSearch" translation missing

### ISSUE THREAD

**Issue by OrlandoRocks** [spree-contrib/spree_i18n#578]
Hi there, i just have a problem in all index from backend where is a quickserch. i have this placeholder input instead:  placeholder="<span class="translation_missing" title="translation missing: es.spree.quick_search" "Quick Search" />"

I hope you can help me with this.

Cheers

**Comment by alepore** [comment]
maybe your language is actually missing that translation? https://github.com/spree-contrib/spree_i18n/search?utf8=%E2%9C%93&q=quick_search

**Comment by OrlandoRocks** [comment]
Yes my language was missing that translation, but now although i have added the error still occurs.
this is the input that appears: 
<input type="text" name="quick_search" id="quick_search" class="form-control js-quick-search" placeholder="&lt;span class=" translation_missing"="" title="translation missing: es.spree.quick_search">
Quick Search" /> "

Thank you for answer, i really appreciate it.

Cheers
